tests/java_definitions.py

```diff
@@ -0,0 +1,24 @@
+import os
+from pathlib import Path
+
+import jpype
+
+REPO_DIR = Path(__file__).parent.parent
+calcite_jars = Path.joinpath(REPO_DIR, "jars/*")
```

Comment:
Calcite or Isthmus?

Reply:
should be isthmus, thanks for the catch!